arbitrator/prover/src/utils.rs

```diff
@@ -97,3 +101,31 @@ impl fmt::Debug for Bytes32 {
         write!(f, "{}", hex::encode(self))
     }
 }
+
+pub fn file_bytes(path: &Path) -> eyre::Result<Vec<u8>> {
+    let mut f = File::open(path)?;
+    let mut buf = Vec::new();
+    f.read_to_end(&mut buf)?;
+    Ok(buf)
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(remote = "Type")]
+enum RemoteType {
+    I32,
+    I64,
+    F32,
+    F64,
+    V128,
+    FuncRef,
+    ExternRef,
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(remote = "TableType")]
+pub struct RemoteTableType {
+    #[serde(with = "RemoteType")]
+    pub element_type: Type,
+    pub initial: u32,
+    pub maximum: Option<u32>,
```

Comment:
We'll need this to be the old table type so that Arbitrator can still deserialize old machines. To test that, you can run `make` on master, switch to your branch, run `make build-node-deps`, and then see if `go test ./system_tests -v -run TestBlockValidatorSimple` still passes. That'll be using the new Arbitrator to attempt to deserialize the old machine.

Reply:
This is now supported. I added a deprecated field to `NameCustomSection` and some `serde_with` stuff for `DeprecatedTableType`, which we'll want to remove when re-initializing the rollup.